### DISCUSSION TITLE
Use flatpak for package management

### DISCUSSION THREAD

**Discussion by ghost** [itsManjeet/avyos#23] [Ideas]
I think flatpak would be a better package management solution than containers because flatpak has sandboxing (although a more Android-like model would be better), is containarized (it can be used on any system with any architecture, C library, etc.), and has a lot of apps packaged already (just check flathub.org). I still love this project, though, but I think it would better without containers.

**Comment by itsManjeet** [comment]
Yes, flatpak is the default package manager is rlxos. Integrated with Software centre to make things easy for new users.
Containers provide additional benefits like softwares that are not available on flathub, or for development environment

**Reply by ghost** [reply]
K then, sorry for creating the issue, love the project btw
